pyaedt/application/Analysis3D.py

```diff
@@ -1277,6 +1277,141 @@ def import_dxf(
         self.oeditor.ImportDXF(vArg1)
         return True
 
+    @pyaedt_function_handler
+    def import_gds_3d(self, gds_file, gds_number, elevation=None, thickness=None, import_method=1):  # pragma: no cover
+        """Import a GDSII file.
+
+        Parameters
+        ----------
+        gds_file : str
+            Path to the GDS file.
+        gds_number : list
+            List of layer numbers to import.
+        elevation : list, optional.
+            List of GDS layer elevation.
```

Comment:
@amichel0205 does it have to be plural elevation -> elevations? and same for thickness -> thicknesses?

Reply:
done